README.md

```diff
@@ -273,7 +317,7 @@ A set of services are started automatically by Soletta in order to do
 initialization, these are listed in
 /usr/lib/soletta/modules/linux-micro/initial-services.
 
-### Flow Based Programming
+## Flow Based Programming
 
 Flow-Based Programming (**FBP**) allows the programmer to express business
```

Comment:
A link to Wikipedia might help as a starting point for the reader to know more about FBP